DigitalOffice/Services/FileService/Validators/AddNewFileValidator.cs

```diff
@@ -10,19 +10,21 @@ public AddNewFileValidator()
         {
             RuleFor(file => file.Name)
                 .NotEmpty()
-                .WithMessage("File must have a name")
-                .MaximumLength(244)
-                .WithMessage("File name is too long")
+                .MaximumLength(244).WithMessage("File name is too long")
```

Comment:
In one line?

Reply:
I also like it better, but I would discuss it explicitly in the chat

Reply:
Yes

Reply:
We have not discussed how this is marked as resolved?

Reply:
Ok

Reply:
Sorry, clicked wrong. I created a poll in Slack